Description: Fix the conflicting declarations of load_file().
Debian-Bug: https://bugs.debian.org/749635
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2015-12-17

--- a/fxc/inc/fxc.h
+++ b/fxc/inc/fxc.h
@@ -141,6 +141,10 @@
 
 extern void div_init() ;
 
+#ifdef BUILDING_FXC
+int load_file (char * filename);
+#endif
+
 #include "offsets.h"
 #include "pslang.h"
 
--- a/fxc/src/main.c
+++ b/fxc/src/main.c
@@ -46,6 +46,7 @@
 #define _(String) String
 #endif
 
+#define BUILDING_FXC
 #include "fxc.h"
 
 #define FXC_VERSION "FXC " VERSION " (" __DATE__ " " __TIME__ ")"
--- a/fxc/src/token.c
+++ b/fxc/src/token.c
@@ -37,6 +37,7 @@
 #define _(String) String
 #endif
 
+#define BUILDING_FXC
 #include "fxc.h"
 #include "messages.c"
 
@@ -568,9 +569,6 @@
 	fprintf (stderr, ")\n") ;
 }
 
-extern void load_file (const char * filename) ;
-
-
 void token_next ()
 {
 	static char buffer[1024] ;
